server/graphql/v2/query/CryptoExchangeRateQuery.js

```diff
@@ -0,0 +1,24 @@
+import { GraphQLNonNull, GraphQLString } from 'graphql';
+
+import { exchangeRateQuery } from '../../../lib/currency';
+import { Currency } from '../enum';
+import { CryptoCurrency } from '../enum/Currency';
+
+const CryptoExchangeRateQuery = {
```

Comment:
Why do we need something specific to crypto? Could we consider something generic instead: exchangeRateQuery

Reply:
It's overly specific, if it's in GraphQL API v2, I want something semantically clean and generic.

Reply:
I've refactored this to `ExchangeRateQuery`.